src/eckit/log/SysLogTCPTarget.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * (C) Copyright 1996- ECMWF.
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ * In applying this licence, ECMWF does not waive the privileges and immunities
+ * granted to it by virtue of its status as an intergovernmental organisation nor
+ * does it submit to any jurisdiction.
+ */
+
+/// @file   SysLogTCPTarget.h
+/// @author Metin Cakircali
+/// @date   July 2025
+
+#include "eckit/log/SysLog.h"
+#include "eckit/log/TCPTarget.h"
+
+namespace eckit {
+
+namespace net {
+class TCPSocket;
+}
+
+//----------------------------------------------------------------------------------------------------------------------
+
+/// sends SysLog (RFC 5424 format) logs over TCP
+class SysLogTCPTarget : public TCPTarget {
+
+    /// Example SysLog:
```

Comment:
Not sure that this example is correct.

For initialising the Target, the content of "message" is irrelevant.

And for sending the messages, the user no longer needs to do this.

Reply:
side affect of different mode that we use this for